tutorindigo/templates/indigo/lms/static/js/dark-theme.js

```diff
@@ -1,8 +1,34 @@
 $(document).ready(function() {
     'use strict';
 
-    const enableDarkTheme = {% if INDIGO_ENABLE_DARK_THEME %}true{% else %}false{% endif %};
-    if (enableDarkTheme){
-        $('body').addClass("indigo-dark-theme");
+    {% if INDIGO_THEME_COOKIE_NAME %}
+    function applyThemeOnPage(){
+      const theme = $.cookie("{{ INDIGO_THEME_COOKIE_NAME }}");
+      $('body').toggleClass("indigo-dark-theme", theme === 'dark');
     }
+
+    function setThemeToggleBtnState(){
+      const theme = $.cookie("{{ INDIGO_THEME_COOKIE_NAME }}");
+      $("#toggle-switch-input").prop("checked", theme === 'dark');
+    }
+    
+    function toggleTheme(){
+      const themeValue = $.cookie("{{ INDIGO_THEME_COOKIE_NAME }}") === 'dark' ? 'light' : 'dark';
+      $.cookie("{{ INDIGO_THEME_COOKIE_NAME }}", themeValue, { domain: window.location.hostname, expires: 7, path: '/' });
```

Comment:
What happens after 7 days? If I understand correctly, the user needs to click the toggle button again. This is not a great UX. Instead, we should use a longer expiry date, which should be updated (pushed back in the future) each time the user loads the page.